ext/tidy/tidy.c

```diff
@@ -379,24 +379,28 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_config_count, 0, 0, 1)
     ZEND_ARG_INFO(0, object)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_getopt, 0, 0, 1)
+ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_getopt, 0, 0, 2)
 	ZEND_ARG_INFO(0, option)
+	ZEND_ARG_INFO(1, object)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_root, 0)
+ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_root, 1)
+	ZEND_ARG_INFO(0, object)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_html, 0)
+ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_html, 1)
+	ZEND_ARG_INFO(0, object)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_head, 0)
+ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_head, 1)
+	ZEND_ARG_INFO(0, object)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_get_body, 0, 0, 1)
 	ZEND_ARG_INFO(0, tidy)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO(arginfo_tidy_construct, 0)
+ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_construct, 0, 0, 0)
```

Comment:
This change can go against PHP-7.1 branch. The others, there're BC. See #3114

Reply:
I don't believe that this is the case. The changes I made in this PR (Apart from the signature of __construct()) **only** affect **global functions**, and don't affect method signatures.

Searching for `'arginfo_tidy_get_head|arginfo_tidy_get_html|arginfo_tidy_get_root|arginfo_tidy_getopt'`

```
342:ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_html_ver, 0)
363:ZEND_BEGIN_ARG_INFO_EX(arginfo_tidy_getopt, 0, 0, 1)
367:ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_root, 0)
370:ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_html, 0)
373:ZEND_BEGIN_ARG_INFO(arginfo_tidy_get_head, 0)
382:    PHP_FE(tidy_getopt,             arginfo_tidy_getopt)
394:    PHP_FE(tidy_get_html_ver,       arginfo_tidy_get_html_ver)
404:    PHP_FE(tidy_get_root,           arginfo_tidy_get_root)
405:    PHP_FE(tidy_get_head,           arginfo_tidy_get_head)
406:    PHP_FE(tidy_get_html,           arginfo_tidy_get_html)
```

This affects global functions (PHP_FE), while the PR you linked affects methods (PHP_ME)

```
104:ZEND_BEGIN_ARG_INFO_EX(arginfo_pdostatement_setfetchmode, 0, 0, 1)
2173:   PHP_ME(PDOStatement, setFetchMode,      arginfo_pdostatement_setfetchmode,      ZEND_ACC_PUBLIC)
```